src/askscript/__tests__/03-lambda/program07b-chain_lambda.ask

```diff
@@ -1,8 +1,3 @@
 ask {
-  score :times(2) :plus(1) :toString :call(fun(s:string): string {
-	  const double = fun(s:string):string {
-		  s:concat(s)
-		}
-	  s:double // same as s:double()
-	}) :toInt :divideBy(6) :ceil
+  score :times(2)
```

Comment:
This test no longer tests lambda functions, please fix.

Reply:
https://github.com/xFAANG/askql/pull/198